website-1.x/data/users.js

```diff
@@ -84,6 +84,13 @@ module.exports = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    caption: 'ConfigCat',
+    image: '/img/users/configcat.svg',
+    infoLink: 'https://github.com/configcat',
```

Comment:
Hi there, thanks for usiing Docusaurus.

I think the `infoLink` shouldn't be the github link but the actual site using Docusaurus.

Reply:
Hi Endi, thanks for the quick review.

I have updated the info link to the actual site using Docusaurus.